public/docs/ts/latest/quickstart.jade

```diff
@@ -1,381 +1,42 @@
++makeExample('index.1.html','my-app','index.html (body)')(format='.')
 :marked
-  Try changing the message in <span ngio-ex>app/app.component.ts</span> to "Hello Again Angular!".
-
-block server-watching
+  The `template` property defines a message inside an `<h1>` header.
+  The message starts with "Hello" and ends with `{{name}}` 
+  which is an Angular [interpolation binding](guide/displaying-data.html) expression.
+  At runtime, Angular replaces `{{name}}` with the value of the component's `name` property. 
+  Change the component class's `name` property from `'Angular'` to `'World'` and see what happens.
+  
+  Binding is one of many Angular features you'll discover in this documentation.
+  
+.l-sub-section
   :marked
-    The TypeScript compiler and `lite-server` will detect your change, recompile the TypeScript into JavaScript,
-    refresh the browser, and display your revised message.
-
-    Close the terminal window when you're done to terminate both the compiler and the server.
-
-.l-main-section
-h2#wrap-up Wrap up and next steps
-:marked
-  The final project folder structure looks like this:
-block project-file-structure
-  .filetree
-    .file angular-quickstart
-    .children
-      .file app
-      .children
-        .file app.component.ts
-        .file app.module.ts
-        .file main.ts
-      .file node_modules ...
-      .file index.html
-      .file package.json
-      .file styles.css
-      .file systemjs.config.js
-      .file tsconfig.json
-:marked
-  To see the file contents, open the <live-example></live-example>.
+    ### Next step
+    Start [learning Angular](guide/learning-angular.html).
 
 .l-main-section
-:marked
-  ## What next?
-  This first application doesn't do much. It's basically "Hello, World" for Angular.
-
-  You wrote a little Angular component, created a simple `index.html`, and launched with a
-  static file server.
-
-+ifDocsFor('ts')
-  :marked
-    You also created the basic application setup that you'll re-use for other
-    sections in this guide. From here, the changes you'll make in the
-    `package.json` or `index.html` files are only minor updates to add libraries or some css
-    stylesheets. You also won't need to revisit module loading again.
-:marked
-  To take the next step and build a small application that demonstrates real features that you can
-  build with Angular, carry on to the [Tour of Heroes tutorial](./tutorial)!
+a#typescript
+:marked
+  ## TypeScript
+  This example is written in <a href="http://www.typescriptlang.org/" target="_blank" title="TypeScript">TypeScript</a>,
+  a strongly-typed, super-set of the latest JavaScript. 
+  TypeScript turns into JavaScript that runs on any modern browser.
+  Most developers find it delightful which is why most Angular developers write applications in TypeScript.
+  You can write your application in [other versions of JavaScript](cookbook/ts-to-js.html) if you prefer.
```

Comment:
It would be great to plug in Dart here too! How about
> You can write your application in [other versions of JavaScript](cookbook/ts-to-js.html), or [Dart](../../dart/latest) if you prefer.